plugins/inputs/conntrack/conntrack.go

```diff
@@ -42,6 +43,10 @@ var dfltFiles = []string{
 	"nf_conntrack_count",
 	"nf_conntrack_max",
 }
+var (
+	all    = false
+	perCPU = false
+)
```

Comment:
Any reason you left these outside the Conntrack struct?

Reply:
I don't like those bool variables, but I can't think of better way how to do this. With variables inside the `Conntrack` struct users will be able to use the directly, right?

Reply:
Hopefully it's better now.